gravity-forms/gw-cache-buster.php

```diff
@@ -184,9 +184,13 @@ public function ajax_get_form() {
 			$form_id = isset( $_GET['form_id'] ) ? absint( $_GET['form_id'] ) : 0;
 		}
 
-		$atts = json_decode( rgpost( 'atts' ), true );
-
-		gravity_form( $form_id, filter_var( rgar( $atts, 'title', true ), FILTER_VALIDATE_BOOLEAN ), filter_var( rgar( $atts, 'description', true ), FILTER_VALIDATE_BOOLEAN ), false, rgar( $atts, 'field_values' ), true /* default to true; add support for non-ajax in the future */, rgar( $atts, 'tabindex' ) );
+		$atts         = json_decode( rgpost( 'atts' ), true );
+		$field_values = rgar( $atts, 'field_values' );
+		// GF expects an associative array for field values. Parse them before passing it on.
+		if ( $field_values ) {
+			parse_str( $field_values, $field_values );
```

Comment:
Can this be replaced with https://developer.wordpress.org/reference/functions/wp_parse_args/?

Reply:
@claygriffiths `wp_parse_args()` performs a few checks `is_object()`, `is_array()` and finally ends up using `wp_parse_str()` for strings which ends up using PHP's built-in `parse_str()` and applying a `wp_parse_str` filter.

Do we need this filter to fire or is this direct call acceptable?

Reply:
I believe replacing these four lines with...

```php
$field_values = wp_parse_args( rgar( $atts, 'field_values' ) );
```

... should do the trick! Let me know if not.

Reply:
@claygriffiths Done. Merging and sending out.